osu.Game/Overlays/MedalSplash/DrawableMedal.cs

```diff
@@ -112,7 +112,7 @@ public DrawableMedal(Medal medal)
 
             medalContainer.OnLoadComplete = d =>
             {
-                unlocked.Position = new Vector2(0f, medalContainer.DrawSize.Y / 2 + 10);
+                unlocked.Position = new Vector2(0f, medalContainer.DrawSize.Y / 2 + 110);
```

Comment:
There's something else going here, `medalContainer` has a `DrawSize` of 0 at this point whereas its children to have valid sizes, so autosize is delayed for some reason.